src/libraries/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/OSPlatform.cs

```diff
@@ -7,25 +7,14 @@ namespace System.Runtime.InteropServices
     {
         private readonly string _osPlatform;
 
-        public static OSPlatform Android { get; } = new OSPlatform("ANDROID");
-
         public static OSPlatform Browser { get; } = new OSPlatform("BROWSER");
 
         public static OSPlatform FreeBSD { get; } = new OSPlatform("FREEBSD");
 
         public static OSPlatform Linux { get; } = new OSPlatform("LINUX");
 
-        public static OSPlatform macOS { get; } = new OSPlatform("MACOS");
-
-        [System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)] // superseded by macOS
         public static OSPlatform OSX { get; } = new OSPlatform("OSX");
 
-        public static OSPlatform iOS { get; } = new OSPlatform("IOS");
-
-        public static OSPlatform tvOS { get; } = new OSPlatform("TVOS");
-
-        public static OSPlatform watchOS { get; } = new OSPlatform("WATCHOS");
-
         public static OSPlatform Windows { get; } = new OSPlatform("WINDOWS");
 
         internal bool IsCurrent { get; } // this information is cached because it's frequently used
```

Comment:
Do we still need this change we made in 5.0 now that we are backing out the other changes?

Reply:
Good catch. OK with me to remove it.